src/include/sof/drivers/edma.h

```diff
@@ -119,4 +119,9 @@
 #define IMX8ULP_DMAMUX2_SAI6_RX	71
 #define IMX8ULP_DMAMUX2_SAI6_TX	72
 
+#ifdef CONFIG_IMX8ULP
+uint32_t imx8ulp_dmamux_rx_int;
+uint32_t imx8ulp_dmamux_tx_int;
```

Comment:
please, don't **define** variables in a header. Why do you need them here? Are they used in multiple files? Then please **define** them in one .c file and **declare** them here with `extern`